mod.yaml

```diff
@@ -268,3 +261,36 @@ GameSpeeds:
 		Name: Fastest
 		Timestep: 20
 		OrderLatency: 6
+
+ModContent:
+	InstallPromptMessage: Red Alert 2 requires artwork and audio from the original game.
+	HeaderMessage: The original game content must be copied from an original game disc.
+	Packages:
+		base: Base Game Files
+			TestFiles: ^Content/ra2/ra2.mix, ^Content/ra2/language.mix, ^Content/ra2/multi.mix
```

Comment:
@Mailaender: did you have a specific reason for including this in the metadata block you posted on https://github.com/OpenRA/OpenRA/pull/11375?